src/zope/interface/common/tests/test_collections.py

```diff
@@ -93,6 +94,10 @@ def test_non_iterable_UserDict(self):
         # It's imported because...? Coverage imports it, but why do we have it without
         # coverage?
         'Row',
+        # In Python 3.10 ``array.array`` appears as ``IMutableSequence`` but it
+        # does not provide a ``clear()`` method and it cannot be instantiated
+        # using ``array.array()``.
+        array.array,
```

Comment:
I wondered if here and below it would be good to only add these entries in the described cases (i.e., only on Python 3.10), but it's probably not worth the effort, because it probably just depends on what gets imported when, and that could change in a minor release.

Reply:
Yes, we would guard against Python 3.10+ but older versions do not have `array.array` in the list either.